awx/main/models/credential.py

```diff
@@ -81,6 +81,38 @@ def build_safe_env(env):
     return safe_env
 
 
+def check_gateway_for_user_in_organization(user, organization, requesting_user):
```

Comment:
good to rename gateway -> resource_server

Reply:
changed